server/index.js

```diff
@@ -1,17 +1,16 @@
-// const express = require('express');
-// const db = require('./db');
-// const app = express();
-// const PORT = 8080;
+const express = require('express');
+const db = require('../database/insertDatabase');
+const app = express();
+const PORT = 8080;
 
-// // req.url and the req.method
-// app.use((req, res, next) => {
-//   console.log(`${req.method} request for ${req.url}`);
-//   next();
-// });
-// app.use(express.static('public'));
-// app.use(express.json());
+// req.url and the req.method
+app.use((req, res, next) => {
+  console.log(`${req.method} request for ${req.url}`);
+  next();
+});
+app.use(express.static('public'));
+app.use(express.json());
 
-// //write a get rewuest for it
-// app.get("/", (req, res)=> {
-
-// });
\ No newline at end of file
+app.get( '/server', (req, res)=> {
+  res.send('got the request');
```

Comment:
I'm not sure what this is doing, do you plan to access the database with this route? Perhaps a more descriptive url from the client's perspective /item or /images.